.travis.yml

```diff
@@ -10,13 +10,13 @@ git:
 addons:
   apt:
     sources:
-    - ubuntu-toolchain-r-test
+      - ubuntu-toolchain-r-test
     packages:
-    - gcc-6
-    - g++-6
+      - gcc-6
+      - g++-6
 
 go:
-  - "1.10.x"
+  - '1.10.x'
```

Comment:
Why ' ?

Reply:
actually, my editor automatically reformatted the file and I didn't notice.
But in general, the difference between single quotes and double quotes is you don't need to escape `\`.
I can revert if you wish.

Reply:
👍